README.md

```diff
@@ -35,40 +40,295 @@ These playbooks install and configure Wazuh agent, manager and Elastic Stack.
+
+```ini
+es1 ansible_host=<es1_ec2_public_ip> private_ip=<es1_ec2_private_ip> elasticsearch_node_name=node-1
+es2 ansible_host=<es2_ec2_public_ip> private_ip=<es2_ec2_private_ip> elasticsearch_node_name=node-2
+es3 ansible_host=<es3_ec2_public_ip> private_ip=<es3_ec2_private_ip> elasticsearch_node_name=node-3
+kibana  ansible_host=<kibana_node_public_ip> private_ip=<kibana_ec2_private_ip>
+manager ansible_host=<manager_node_public_ip> private_ip=<manager_ec2_private_ip>
+worker  ansible_host=<worker_node_public_ip> private_ip=<worker_ec2_private_ip>
+
+[odfe_cluster]
+es1
+es2
+es3
+
+[all:vars]
+ansible_ssh_user=vagrant
+ansible_ssh_private_key_file=/path/to/ssh/key.pem
+ansible_ssh_extra_args='-o StrictHostKeyChecking=no'
 ```
 
-2. Copy the `Pipfile` and the `molecule` folder into the root wazuh-ansible directory:
+### Launching playbook
 
+```bash
+ansible-playbook wazuh-odfe-production-ready.yml -i inventory
 ```
-cp wazuh-qa/ansible/wazuh-ansible/* . -R
+
+After the playbook execution, Wazuh UI should be reachable through `https://<kibana_host>:5601`
+
+## Example: single-host environment
+
+### Playbook
+The hereunder example playbook uses the `wazuh-ansible` role to provision a single-host Wazuh environment. This architecture includes all the Wazuh and ODFE components in a single node.
+
+```yaml
+---
+# Single node
+    - hosts: server
+      become: yes
+      become_user: root
+      roles:
+        - role: ../roles/opendistro/opendistro-elasticsearch
+        - role: "../roles/wazuh/ansible-wazuh-manager"
+        - role: "../roles/wazuh/ansible-filebeat-oss"
+        - role: "../roles/opendistro/opendistro-kibana"
+      vars:
+        single_node: true
+        minimum_master_nodes: 1
+        elasticsearch_node_master: true
+        elasticsearch_network_host: <your server host>  
+        filebeat_node_name: node-1
+        filebeat_output_elasticsearch_hosts: <your server host>
+        ansible_ssh_user: vagrant
+        ansible_ssh_private_key_file: /path/to/ssh/key.pem
+        ansible_ssh_extra_args: '-o StrictHostKeyChecking=no'
+        instances:
+          node1:
+            name: node-1       # Important: must be equal to elasticsearch_node_name.
+            ip: <your server host>
 ```
 
-3. Follow these steps for launching the tests. Check the Pipfile for running different scenarios:
+### Inventory file
 
+```ini
+[server]
+<your server host>
+
+[all:vars]
+ansible_ssh_user=vagrant
+ansible_ssh_private_key_file=/path/to/ssh/key.pem
+ansible_ssh_extra_args='-o StrictHostKeyChecking=no'
 ```
-pip install pipenv
-sudo pipenv install
-pipenv run test
-pipenv run agent
+
+### Launching playbook
+
+```bash
+ansible-playbook wazuh-odfe-single.yml -i inventory
 ```
 
+After the playbook execution, Wazuh UI should be reachable through `https://<your server host>:5601`
+
+## Branches
+* `master` branch on correspond to the last Wazuh-Ansible stable version.
```

Comment:
```suggestion
* `master` branch corresponds to the latest Wazuh Ansible changes. It might be unstable.
```

Reply:
Also, it'd be necessary to describe the release branches. (3.13, 4.0, and so)

Reply:
Fixed on 58eb10c6dc74a0a03fc3f5b30ce3eeb996ddd8e3